src/composables/usei18nSlugs.js

```diff
@@ -0,0 +1,16 @@
+export async function useI18nSlugs() {
+    const route = useRoute()
+
+    const { data } = await useAsyncData(`${route.name}-${route.params.slug}-i18n-slugs`, async () => {
+        const i18nRouteConfig = (await import('../../.cache/i18n-slugs.json')).default
+        const i18nSlugs = i18nRouteConfig.find((i18nRoute) => i18nRoute.route === route.name)
+
+        if (!i18nSlugs) {
+            return null
+        }
+
+        return i18nSlugs.slugs.find(localeConfig => localeConfig.find(({ locale, value }) => value === route.params.slug && locale === route.params.language))
```

Comment:
Bit of a long one-liner don't you think?

Reply:
[1daf67c](https://github.com/voorhoede/voorhoede-website/pull/613/commits/1daf67c20e11a35ce58271d684671d00ebe4e0a5) I split it up in multiple lines, but I could also separate it into a function/variables. Let me know what you prefer.